src/lib/elementary/elc_scrolled_entry.c

```diff
@@ -4,194 +4,194 @@
 
 #include <Elementary.h>
 #include "elm_priv.h"
-EINA_DEPRECATED EAPI Evas_Object *
+EINA_DEPRECATED ELM_API Evas_Object *
```

Comment:
This should follow https://docs.microsoft.com/pt-br/cpp/build/exporting-from-a-dll-using-declspec-dllexport?view=vs-2017?

Reply:
I mean, this part specifically:
> To export functions, the __declspec(dllexport) keyword must appear to the left of the calling-convention keyword, if a keyword is specified.

Reply:
In this case, this is not necessary: `EINA_DEPRECATED` is also a `__declspec`.